public/index.html

```diff
@@ -8,11 +8,37 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Excalidraw is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them."
+    />
+    <meta name="image" content="%PUBLIC_URL%/og-image.png" />
+
+    <!-- OpenGraph tags -->
+    <meta property="og:url" content="https://www.excalidraw.com/" />
+    <meta property="og:site_name" content="Excalidraw" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Excalidraw" />
+    <meta
+      property="og:description"
+      content="Excalidraw is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them."
+    />
+    <meta property="og:image" content="%PUBLIC_URL%/og-image.png" />
+
+    <!-- Twitter Card tags -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@Vjeux" />
+    <meta name="twitter:creator" content="@Vjeux" />
+    <meta name="twitter:image" content="%PUBLIC_URL%/og-image.png" />
```

Comment:
could me merge with the property image..

```
<meta property="og:image" name="twitter:image"...
```

Reply:
Sure!